app/models/post.rb

```diff
@@ -1,5 +1,17 @@
 class Post < ApplicationRecord
+  after_save :increment_posts_counter
+
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [OPTIONAL] Same issue as above, please consider.

Reply:
Done. Thanks, @Lameck1 for the insight!